conduit/src/components/Feed.jsx

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect, useDispatch } from "react-redux";
+import { getArticles, postLike, liked } from "../store/actions";
+import { useEffect } from "react";
+import { useSelector } from "react-redux";
+import moment from "moment";
+import { returnState } from "../store/returnState";
+import Loader from "./Loader";
+import Tags from "./Tags";
+import If from "./If";
+// material ui imports
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
+import CardContent from "@material-ui/core/CardContent";
+import CardActions from "@material-ui/core/CardActions";
+import IconButton from "@material-ui/core/IconButton";
+import Typography from "@material-ui/core/Typography";
+import FavoriteIcon from "@material-ui/icons/Favorite";
+import FavoriteBorderIcon from "@material-ui/icons/FavoriteBorder";
+
+const useStyles = makeStyles((theme) => ({
+  loaderDiv: { display: "inline-flex" },
+  loaderP: { fontSize: "20px", lineHeight: "35px", fontWeight: "bold" },
+  root: {
+    width: "90%",
+    marginBottom: "25px",
+    marginLeft: "25px",
+  },
+  media: {
+    height: 0,
+    paddingTop: "56.25%", // 16:9
+  },
+  expand: {
+    transform: "rotate(0deg)",
+    marginLeft: "auto",
+    transition: theme.transitions.create("transform", {
+      duration: theme.transitions.duration.shortest,
+    }),
+  },
+  expandOpen: {
+    transform: "rotate(180deg)",
+  },
+  avatar: {
+    width: "50px",
+    height: "50px",
+    borderRadius: "50%",
+  },
+  container: {
+    display: "grid",
+    gridTemplateColumns: "60% 40%",
+    gap: "0px",
+  },
+  // tags : {'float' : 'right'}
+}));
+
+const Feed = (props) => {
+  //hooks
+  const dispach = useDispatch();
+  const classes = useStyles();
+  const state = useSelector(() => returnState(props));
+  useEffect(() => {
+    props.getArticles(url);
```

Comment:
Why you are using variables before it's declare , these type of constant should be in file called static